waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -93,15 +99,25 @@ migrateDiff projectDir = runPrismaCommandAsDbJob projectDir $ \schema ->
 -- Therefore, this should be checked **after** a command that ensures connectivity.
 migrateStatus :: Path' Abs (Dir ProjectRootDir) -> J.Job
 migrateStatus projectDir = runPrismaCommandAsDbJob projectDir $ \schema ->
-  ["migrate", "status", "--schema", SP.fromAbsFile schema]
+  [ "migrate",
+    "status",
+    "--schema",
+    SP.fromAbsFile schema
+  ]
```

Comment:
If you really like it better in multiple lines fine, but I liked it better in one line, found it easier to read. And since it has nothing to do with this PR maybe we could revert it? Same goes for other couple of functions in this file.